rental.rb

```diff
@@ -0,0 +1,15 @@
+require_relative './book'
+require_relative './person'
+
+class Rental
+  attr_accessor :date, :book, :person
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    @person = person
+
+    book.add_rental(self)
+    person.add_rental(self)
```

Comment:
- [ ] Here, there would be no need to call the `add_rental` method anymore if you have made the changes I suggested in the `add_rental` methods in your `book.rb` and `person.rb` files.
All you have to do now is to push the rental into the `@rentals` on both `Person` and `Book`.
**HINT**
```ruby
def initialize(date, book, person)
  .
  .
  .
  book.rentals << self
  person.rentals << self
end
```